docs/organizations/members.md

```diff
@@ -1,52 +1,65 @@
 # Organization Members
 
-An organization can have multiple members, with limits on members and pending invitations depending on the organization's tier level. The organization's overview screen displays a list of current members, and the `Members` screen provides more details on each member. If you're an Administrator, that screen also displays a list of pending and expired invitations.
+An organization can have multiple members, with limits on members and pending invitations depending on the organization's tier level. The organization's overview screen displays a list of current members, the 'Members' screen allows an administrator to perform quick actions such as removing a member or renewing a pending invite, and the `Edit Member` screen provides more details on each member. If you're an Administrator, that screen also displays a list of pending and expired invitations.
```

Comment:
By the final sentence, it's unclear which screen you're talking about; and, only administrators can access the latter two screens.